src/VisualStudio/Core/Test/Options/CSharpEditorConfigGeneratorTests.vb

```diff
@@ -249,9 +250,9 @@ dotnet_naming_style.begins_with_i.required_suffix =
 dotnet_naming_style.begins_with_i.word_separator = 
 dotnet_naming_style.begins_with_i.capitalization = pascal_case
 "
-                Dim editorConfigOptions = CSharp.Options.Formatting.CodeStylePage.TestAccessor.GetEditorConfigOptions()
-                Dim options = New OptionStore(workspace.GlobalOptions)
-                Dim actualText = EditorConfigFileGenerator.Generate(editorConfigOptions, options, LanguageNames.CSharp)
+                ' Use the default options
```

Comment:
I modified the default options test here to just use the external access EditorConfigGenerator service directly. The service then calls EditorConfigFileGenerator. I could make a new test file somewhere if people want the EditorConfigGenerator and the EditorConfigFileGenerator tested separately. I just wasn't certain as both the csharp and vb tests are written in vb.

Reply:
Seems like no one has a problem with this - resolving!